src/__swaps__/utils/flipAssets.ts

```diff
@@ -0,0 +1,137 @@
+import { SharedValue } from 'react-native-reanimated';
+import { inputKeys, inputMethods, inputValuesType } from '@/__swaps__/types/swap';
+import { valueBasedDecimalFormatter } from '@/__swaps__/utils/decimalFormatter';
+import { niceIncrementFormatter } from '@/__swaps__/utils/swaps';
+import { ExtendedAnimatedAssetWithColors } from '@/__swaps__/types/assets';
+import { divWorklet, equalWorklet, greaterThanWorklet, mulWorklet } from '@/__swaps__/safe-math/SafeMath';
+
+export function getInputValuesForSliderPositionWorklet({
```

Comment:
this is the function that was just directly moved over from the `useSwapInputsController` - no changes